### PR TITLE
doc(release) Release notes for modules 24.04 on prem

### PR DESCRIPTION
**Pull request by pkippes**
map 
anomaly detection
bam
mbi
IT Edition Extensions

## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [x] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 1]
```diff
@@ -21,120 +21,122 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
...
+## Centeon IT Edition Extensions
+
+### 24.04.0
+
+Release date: `April 30, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [API] Added a new endpoint to list shares of playlists.
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx:123]: J'aurais ajouté une sous section "Dashboard Playlists"

File: versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 2]
```diff
@@ -16,125 +16,128 @@ In this chapter, you can find all the changelogs concerning **Centreon Commercia
...
-- [Viewer] Made resource status unique for hosts (include host status in inherited status computation).
-- [Viewer] You can now define a label for a URL shape.
-
-</details>
-
-<details>
-  <summary>Compatibility</summary>
-
-- Compatibility with other 23.10 components.
+- To facilitate smooth transition from MAP Legacy to MAP, we have made sure that features and enhancements already available in previous versions are fully compatible.
```
Comment (cg-tw) [inline versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx:32]:
```suggestion
- To facilitate a smooth transition from MAP Legacy to MAP, we have made sure that features and enhancements already available in previous versions are fully compatible.
```

File: versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 3]
```diff
@@ -16,125 +16,128 @@ In this chapter, you can find all the changelogs concerning **Centreon Commercia
...
-Release date: `October 30, 2023`
+Release date: `April 30, 2024`
 
-<details>
-  <summary>Compatibility</summary>
+<details open>
+  <summary>Enhancements</summary>
 
-- Compatibility with other 23.10 components.
+- [API] Added an endpoint to UPDATE Cloud Token.
```
Comment (cg-tw) [inline versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx:110]:
```suggestion
- [API] Added an endpoint to UPDATE Cloud tokens.
```

File: versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 4]
```diff
@@ -16,125 +16,128 @@ In this chapter, you can find all the changelogs concerning **Centreon Commercia
...
+Release date: `April 30, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [API] Added a new endpoint to list shares of playlists.
+- [API] Added a new endpoint to share playlists.
+- [API] Added a new endpoint to create a playlist of dashboards.
+- [API] Added a new endpoint to list playlists of dashboards.
+- [API] Added a new endpoint to revoke the share of a contact group on a playlist.
```
Comment (cg-tw) [inline versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx:128]:
```suggestion
New playlist feature: dashboard playlists allow you to display several dashboards sequentially without any interaction. Ideal for broadcast on a TV screen. 
	
You can now:
- create dashboard playlists.
- display the list of dashboard playlists.
- share playlists.
- display the list of playlist shares.
- revoke the sharing of a playlist to a contact group.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 5]
```diff
@@ -15,126 +15,131 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
+Release date: `April 30, 2024`
 
-<details>
-  <summary>Compatibility</summary>
+<details open>
+  <summary>Enhancements</summary>
 
-- Compatibility with other 23.10 components.
+- [API] Added an endpoint to UPDATE Cloud tokens.
+- [API] Added an endpoint to GET Business Activities.
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx:110]:
```suggestion
```

Comment (lpinsivy) [reply i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx:110]: already present in BAM

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 6]
```diff
@@ -15,126 +15,131 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
 
 <details>
   <summary>Compatibility</summary>
 
-- Compatibility with other 23.10 components.
+- Compatibility with other 24.04 components.
 
 </details>
 
 ## Centreon Monitoring Connectors Manager (formerly Plugin Packs Manager)
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx:74]:
```suggestion
## Centreon Monitoring Connectors Manager
```

File: versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 7]
```diff
@@ -16,125 +16,131 @@ In this chapter, you can find all the changelogs concerning **Centreon Commercia
...
 
 <details>
   <summary>Compatibility</summary>
 
-- Compatibility with other 23.10 components.
+- Compatibility with other 24.04 components.
 
 </details>
 
 ## Centreon Monitoring Connectors Manager (formerly Plugin Packs Manager)
```
Comment (lpinsivy) [inline versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx:75]:
```suggestion
## Centreon Monitoring Connectors Manager
```

File: versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx [hunk 8]
```diff
@@ -16,125 +16,131 @@ In this chapter, you can find all the changelogs concerning **Centreon Commercia
...
+Release date: `April 30, 2024`
 
-<details>
-  <summary>Compatibility</summary>
+<details open>
+  <summary>Enhancements</summary>
 
-- Compatibility with other 23.10 components.
+- [API] Added an endpoint to UPDATE Cloud tokens.
+- [API] Added an endpoint to GET Business Activities.
```
Comment (lpinsivy) [inline versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx:111]:
```suggestion
```

**Review by lpinsivy — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by lpinsivy — APPROVED** [review]

**Review by nmagniezCentreon — APPROVED** [review]

**Pull request merged**